index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Coin Toss Solo Build</title>
+</head>
+<link rel="stylesheet" href="styles.css">
+<body>
+    <header><h1>Panda Flip Game</h1></header>
+    <main>
+        <p class= "scoreboard">
+                <span id="wins">Wins: 0</span>
+                <span id="losses">Losses: 0</span>
+        </p>
+        <p>
+            <img id="panda" src="assets/pandaButt2.jpeg">
+        </p>
+                Give it a whirl. 
+        </p>
+        <p id="message"></p>
+        <p>
```

Comment:
Be sure to add attributes to your labels. The attribute `for="heads"` would be applied to the input with the id of heads.